__tests__/frontend-proxy.ts

```diff
@@ -73,22 +73,41 @@ describe('handleRequest()', () => {
     })
   })
 
-  describe('prepends language code when FRONTEND_SUPPORT_INTERNATIONALIZATION is "true"', () => {
-    test.each(Object.values(LanguageCode))(
-      'language code = %p',
-      async (lang) => {
-        global.FRONTEND_SUPPORT_INTERNATIONALIZATION = 'true'
+  describe('when FRONTEND_SUPPORT_INTERNATIONALIZATION is "true"', () => {
+    describe('prepends language code to path when backend is frontend', () => {
+      test.each([LanguageCode.En, LanguageCode.En])(
```

Comment:
That doesn't look useful ^^
```suggestion
      test.each([LanguageCode.En, LanguageCode.De])(
```